CHANGELOG.md

```diff
@@ -15,6 +27,45 @@ This can be a breaking change on windows if we rely on a stack snapshot containi
 as we encounter [this issue](https://github.com/haskell/win32/issues/193).
 In this case, to ensure the toolchain `Win32` library is used (instead of trying to compile a new one),
 it can be droped from a custom stack snapshot as in the [./stackage_snapshot.yaml](./stackage_snapshot.yaml) file.
+* Handle CPU value darwin_arm64 (see [#1739])
+* Support Bazel 4.2.2 (see [#1695])
```

Comment:
I think the [Bazel 5 one](https://github.com/tweag/rules_haskell/pull/1781) only needs a `merge-queue` label at this point. Would be nice to be able to add that to the list.

Reply:
:+1: done